app/src/main/java/com/android/streetworkapp/ui/park/ParkOverview.kt

```diff
@@ -70,6 +70,8 @@ fun ParkOverviewScreen(
 
   currentPark.value?.let { eventViewModel.getEvents(it) }
 
+  parkViewModel.updateCurrentParkNameNominatim()
```

Comment:
I’m not sure of the usefulness of this function here. Every time a park is selected, it call the ViewModel function.

Wouldn’t it be simpler to call this function just once when a park is created, and then delegate this task to the ParkViewModel?

Reply:
I see why it was implemented like that then, I mark the comment as resolved, thank you!